src/track/index.js

```diff
@@ -0,0 +1,7 @@
+import Track from './track'
+import VideoTrack from './video-track'
+import AudioTrack from './audio-track'
+import TextTrack from './text-track'
+import TRACK_TYPES from './track-types'
+
```

Comment:
The name of the file is too generic (although it under track folder).
Maybe we need convention to things like that for similar use in the future.
Something like 'track-export.js'.

Having saying that, I think this file is unnecessary and we just need to copy this content to playkit.js